universal-application-tool-0.0.1/app/views/ViewUtils.java

```diff
@@ -25,4 +26,13 @@ Tag makeLocalJsTag(String filename) {
         .withSrc(assetsFinder.path("javascripts/" + filename + ".js"))
         .withType("text/javascript");
   }
+
+  /**
+   * Generates an HTML link tag for loading the CSS file found at public/stylesheets/[filename].css.
+   */
+  Tag makeLocalCssTag(String filename) {
+    return link()
+        .withHref(assetsFinder.path("stylesheets/" + filename + ".css"))
+        .withRel("stylesheets");
+  }
```

Comment:
question (blocking): confirm the stylesheet actually loads in the browser

Reply:
Wow - great catch. I attempted to load common.css in the J2HtmlView. The link tag is there, but the stylesheet isn't. Investigating.

Reply:
Turns out I needed `rel="stylesheet"` instead of `rel="stylesheets"` 😝 

Works now!
![Screen Shot 2021-02-19 at 10 05 51 AM](https://user-images.githubusercontent.com/5732310/108543573-1d7a0c80-729a-11eb-857c-0ceea36790a3.png)

![Screen Shot 2021-02-19 at 10 06 10 AM](https://user-images.githubusercontent.com/5732310/108543590-210d9380-729a-11eb-94d0-29e68d531b24.png)

Reply:
😁

Well thanks to you writing `makeLocalCssTag` it won't happen to anyone else!